### PR TITLE
Fix modal backdrop

### PR DESCRIPTION
**Pull request by ma12-co**
Fix: #748

- [x] Make default backdrop light
- [x] Add prop to toggle black backdrop
- [x] Make modal box shadow softer

Here's how it looks: 

Screenshot from 2019-11-27 15-42-15

### CODE REVIEW

File: src/components/Modal/Modal.vue [hunk 1]
```diff
@@ -459,7 +467,11 @@ $header-size: 50px;
 	display: block;
 	width: 100%;
 	height: 100%;
-	background-color: rgba(0, 0, 0, .92);
+	background-color: rgba(0, 0, 0, .5);
+}
+
+.dark {
```
Comment (skjnldsv) [inline src/components/Modal/Modal.vue]: bem `--dark`

File: src/components/Modal/Modal.vue [hunk 2]
```diff
@@ -643,7 +655,7 @@ $header-size: 50px;
 		transition: transform 300ms ease;
 		border-radius: var(--border-radius-large);
 		background-color: var(--color-main-background);
-		box-shadow: 0 2px 8px rgba(0, 0, 0, .33);
+		box-shadow: 0 0 40px rgba(0, 0, 0, .2);
```
Comment (skjnldsv) [inline src/components/Modal/Modal.vue]: why changing the box shadow?

Comment (ma12-co) [reply src/components/Modal/Modal.vue]: to make it less harsh, with the opacity at 90% it was not a problem but after bringing that down It was too strong IMO

Comment (skjnldsv) [reply src/components/Modal/Modal.vue]: Could you add screenshots of the comparison pls :)

Comment (ma12-co) [reply src/components/Modal/Modal.vue]: Before: 
Screenshot from 2019-11-27 16-06-54

After:
Screenshot from 2019-11-27 16-07-36

File: src/components/Modal/Modal.vue [hunk 3]
```diff
@@ -60,7 +60,10 @@ export default {
 </docs>
 <template>
 	<transition name="fade">
-		<div ref="mask" class="modal-mask" @click="handleMouseMove"
+		<div ref="mask"
+			class="modal-mask"
+			:class="{ 'modal-mask--dark' : dark }"
```
Comment (skjnldsv) [inline src/components/Modal/Modal.vue]:
```suggestion
			:class="{ 'modal-mask--dark': dark }"
```

**Pull request merged**